internal/component/beyla/ebpf/beyla_linux.go

```diff
@@ -338,7 +400,7 @@ func (args Metrics) hasAppFeature() bool {
 
 func (args Metrics) Validate() error {
 	validInstrumentations := map[string]struct{}{
-		"*": {}, "http": {}, "grpc": {}, "redis": {}, "kafka": {}, "sql": {},
+		"*": {}, "http": {}, "grpc": {}, "redis": {}, "kafka": {}, "sql": {}, "gpu": {}, "mongo": {},
```

Comment:
Does it make sense to make this list reusable instead of copying it in few places?